cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -1005,8 +1005,13 @@ if("${MAKE}" STREQUAL "")
   endif()
 endif()
 
-# Args for external projects using make.
-set(MAKE_BUILD_ARGS "-j${NPROC}")
+# Args for external projects using make
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.28")
+  #Prevent 'bad file descriptor' error
```

Comment:
```suggestion
  # Prevent 'bad file descriptor' error
```

Reply:
Can you add a reference to the GH issue?